src/chip/gates/nor.rs

```diff
@@ -0,0 +1,223 @@
+use std::time::Duration;
+
+use crate::{
+    chip::{ChipBuilder, ChipRunner, ChipType, Pin, PinId, PinType},
+    generate_chip, State,
+};
+
+/// # A chip with 4 bundled "OR" gates
```

Comment:
This a little nit pick but that should probably say NOR and not OR

Reply:
Right, copy-pasting typo, Fixing that asap